templates/settings.yml

```diff
@@ -0,0 +1,104 @@
+# These settings are synced to GitHub by https://probot.github.io/apps/settings/
```

Comment:
Same configuration that was done for https://github.com/playframework/play-scala-starter-example/pull/121